spec/features/social_features_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+
+feature "Social networking" do
+    given(:jane) { Fabricate(:user) }
+    given(:joe) { Fabricate(:user) }
+    given(:tv_comedies) { Fabricate(:category, name: 'TV Comedies') }
+
+  scenario "following a user" do 
+    futurama = Fabricate(:video, title: 'Futurama', category: tv_comedies)
+    jane_review = Fabricate(:review, video: futurama, user: jane)
+    
+    sign_in(joe)
+    find("a[href='/videos/#{futurama.id}']").click 
+    click_on jane_review.user.full_name
+    click_on 'Follow'
+    expect(current_path).to eq(people_path)
```

Comment:
[This](https://github.com/dcousette/myflix/commit/b16f86fa789879fea1900f63210049af060c5254#diff-34b2c7f0f6901aa6ea2fecfc2caccfd0R21) was a better expectation. The problem you have now is that it doesn't really tell you if jane was followed, only that you were taken to the `people_path`.

Here is the idea. You are giving directions and trying to get them to a location. What you had before could be. Go north on the road, turn left at the scarecrow (make sure the scarecrow's face is red and one eye is missing), pick up Jane at the purple house (make sure the paint is peeling) and take her to the hoedown where the tractor is (make sure jane gets on the ride).

The problem you had was that you don't really care about the scarecrow's face or missing eye. The farmer may have fixed him but it is unlikely he is gone altogether and so we'll use him as a guide post, same for the purple house. What you miss in this version is the final (and actually important) part--did jane get on the tractor ride? You are basically confirming there is a ride, but this doesn't really indicate if jane is on it or not. Writing a test is all about understanding what it is you are testing, and then actually testing it.